Source/WebCore/page/writing-tools/WritingToolsController.mm

```diff
@@ -568,11 +567,11 @@
 
     auto rangeAfterReplace = characterRange(sessionRange, *selectionRange);
 
-    compositionSessionDidFinishReplacement(rangeAfterReplace, attributedText.string);
+    compositionSessionDidFinishReplacement(sourceAnimationUUID, destinationAnimationUUID, rangeAfterReplace, attributedText.string);
```

Comment:
when is there ever more than one destination / source animation ongoing at the same time?

Reply:
I'm not sure I understand the question. We need these IDs so that the destination animation can run properly on iOS.